### ISSUE TITLE
getSubBlocks should not be SideOnly.CLIENT.

### ISSUE THREAD

**Issue by KingLemming** [SlimeKnights/Mantle#103]
Basically as topic. I don't know how many of your blocks this is in, but I know it's in EnumBlock at least.

I'm trying to track down potential weirdness with Natura and while I can't say for certain this is the cause...it could be? :)

**Comment by bonii-xx** [comment]
You are correct, it's from the days of clientside-only-creative-items-thing. Curious why you are iterating over it though? I don't think it's a good idea to use it for any non-display purposes.

**Comment by KingLemming** [comment]
Because when you OreDict things using the Wildcard value instead of individually, you leave me no choice.

Basically, to figure out Natura's logs from the "logWood" list to create the Sawmill recipes, I have to call getSubBlocks to get each actual log.

**Comment by weirleader** [comment]
That did the trick -- thanks!
